BTrees/_compat.py

```diff
@@ -23,7 +24,17 @@
 
     int_types = int, long
     xrange = xrange
-    cmp = cmp
+    cmp_ = cmp
+    def cmp(x, y):
```

Comment:
Maybe we should call this (and the one below) something else to emphasize that it's not the standard Python `cmp`?